Classifier_Klasse.py

```diff
@@ -32,12 +51,17 @@ def predict_model(self, train):
         df_class = distanz_ironic.gt(distanz_notironic).astype(int)
         return df_class.replace({0: "Ironic", 1: "Not Ironic"})
 
-        
-def main():
+@click.command()
+@click.argument('data', required=True)
 
-    DATAPATH = "/Users/Natasha/Library/Mobile Documents/com~apple~CloudDocs/Универ/Sommersemester 21/Prog 2/DATA/Data_train_features.csv"
-    df = pd.read_csv(DATAPATH)
-    train_model = Classifier(df).train_model()
+def main(data):
+
+    df = pd.read_csv(data)
+    print("Available features: ", list(df.columns)[1:])
+    features_for_train = input("Please enter features for model training by whitespace without quotes and commas: ")
+    if type(features_for_train) != str:
```

Comment:
Typkontrollen realisiert man am besten so:
`if not isinstance(features_for_train, str):`